snuba/web/query.py

```diff
@@ -241,6 +241,11 @@ def parse_and_run_query(
         request.extensions["timeseries"]
     )
 
+    if (
+        request.query.get_sample() is not None and request.query.get_sample() != 1.0
```

Comment:
Someone is explicitly passing sample = 1 and turbo to achieve no final but no sampling.